Tests/DefaultsTests/DefaultsRangeTests.swift

```diff
@@ -0,0 +1,221 @@
+import Foundation
+import Defaults
+import XCTest
+
+private struct Date {
```

Comment:
I would name it `CustomDate` so it's not confused with the built-in `Date` when reading the code.